liblangutil/ParserBase.cpp

```diff
@@ -102,7 +102,7 @@ void ParserBase::expectTokenOrConsumeUntil(Token _value, string const& _currentN
 		SourceLocation errorLoc = currentLocation();
 		int startPosition = errorLoc.start;
 		while (m_scanner->currentToken() != _value && m_scanner->currentToken() != Token::EOS)
-			m_scanner->next();
+			advance();
 
 		string const expectedToken = ParserBase::tokenName(_value);
 		if (m_scanner->currentToken() == Token::EOS)
```

Comment:
The same with this. There are helpers on `ParserBase` to avoid using `m_scanner`.